pypesto/visualize/profiles.py

```diff
@@ -215,6 +253,12 @@ def profiles_lowlevel(
         # if we have empty profiles and more axes than profiles: skip
         if n_plots != n_fvals and fval is None:
             continue
+        # If we use colors from profiler_result.color_path,
+        # we need to take the color path of each profile
+        if isinstance(color, list) and isinstance(color[i_plot], np.ndarray):
+            i_color = color[i_plot]
+        else:
+            i_color = color
```

Comment:
Just a personal preference but overall we tend to use `i_something` as an index so this could instead be `color_i`

Reply:
Like it, it's nicer